train.py

```diff
@@ -372,6 +373,7 @@ def loss_fn(pred, labels):
                     "loss_metrics/global_avg_loss": global_avg_loss,
                     "loss_metrics/global_max_loss": global_max_loss,
                     "throughput(tps)": tps,
+                    "tflops(s)": tflops,
```

Comment:
```suggestion
                    "tflops": tflops,
```
sorry my previous suggestion doesn't make sense -- in TFLOPs, "s" already stands for per second, so I think using `tflops` alone is good enough